src/index.js

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
 import MapScreen from './Components/MapScreen/MapScreen';
+import * as firebase from 'firebase';
```

Comment:
Do we need to import * here?

Reply:
Looks like most sources just use `import firebase from 'firebase'` so I'll give that a shot instead.

Reply:
Merging with the understanding that Cable will fix this. This is so we can all base off of the meat of the PR.